tests/integration/test_api.py

```diff
@@ -39,7 +39,7 @@ def test_get_listens(self):
         # This sleep allows for the influx subscriber to take its time in getting
         # the listen submitted from redis and writing it to influx.
         # Removing it causes an empty list of listens to be returned.
-        time.sleep(10)
+        time.sleep(15)
```

Comment:
Ah, then we have 2 problems, because the test failures i saw were because the api container had not come up yet. :(